datadog/resource_datadog_monitor.go

```diff
@@ -509,7 +508,7 @@ func resourceDatadogMonitorRead(d *schema.ResourceData, meta interface{}) error
 
 	// Ignore any timestamps in the past that aren't -1 or 0
 	for k, v := range configSilenced {
-		if v.(int) < int(time.Now().Unix()) && v.(int) != 0 && v.(int) != -1 {
+		if v.(int) < int(providerConf.Now().Unix()) && v.(int) != 0 && v.(int) != -1 {
```

Comment:
Not sure I fully understand what this change addresses?

Reply:
problem in replaying tests, since we do a match on the body now. So we need to be able to set the frozen clock in the test, this allows us to do that

Reply:
Ahh ok, so no functional user difference, just for tests 👍